### PR TITLE
rwbox.is-an.app

### PR DESCRIPTION
**Pull request by ourstorycomic**
### Link to Website

Link: https://ourstorycomic.github.io/rw-box

### CODE REVIEW

File: domains/rwbox.is-an.app.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "$schema": "../schemas/domain.schema.json",
+  "description": "is a project of RW Studio about an application that allows viewers to experience comic, movies and TV shows with better quality, without ads on internet connected devices, rating and commenting on our projects, download our comics, movies and TV Shows easily.",
+  "domain": "is-an.app",
+  "subdomain": "rwbox",
+  "owner": {
+    "repo": "https://github.com/ourstorycomic/rw-box",
+    "gmail": "kaz442197@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://ourstorycomic.github.io/rw-box",
```
Comment (tarampampam) [inline domains/rwbox.is-an.app.json:11]:
```suggestion
    "CNAME": "hourstorycomic.github.io.",
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/rwbox.is-an.app.json (hunk 1 above)
Comment (tarampampam) [reply domains/rwbox.is-an.app.json:11]: ping @ourstorycomic

File: domains/rwbox.is-an.app.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "is a project of RW Studio about an application that allows viewers to experience comic, movies and TV shows with better quality, without ads on internet connected devices, rating and commenting on our projects, download our comics, movies and TV Shows easily.",
+  "domain": "is-an.app",
+  "subdomain": "rwbox",
+  "owner": {
+    "repo": "https://github.com/ourstorycomic/rw-box",
+    "gmail": "kaz442197@gmail.com"
```
Comment (tarampampam) [inline domains/rwbox.is-an.app.json:8]:
```suggestion
    "email": "kaz442197@gmail.com"
```

File: domains/rwbox.is-an.app.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "$schema": "../schemas/domain.schema.json",
+  "description": "is a project of RW Studio about an application that allows viewers to experience comic, movies and TV shows with better quality, without ads on internet connected devices, rating and commenting on our projects, download our comics, movies and TV Shows easily.",
+  "domain": "is-an.app",
+  "subdomain": "rwbox",
+  "owner": {
+    "repo": "https://github.com/ourstorycomic/rw-box",
+    "email": "kaz442197@gmail.com"
+  },
+  "record": {
+    "CNAME": "hourstorycomic.github.io.",
```
Comment (tarampampam) [inline domains/rwbox.is-an.app.json:11]:
```suggestion
    "CNAME": "hourstorycomic.github.io."
```

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
Done:

```
$ dig @1.1.1.1 +noall +answer rwbox.is-an.app
rwbox.is-an.app.	246	IN	CNAME	hourstorycomic.github.io.
hourstorycomic.github.io. 3546	IN	A	185.199.109.153
hourstorycomic.github.io. 3546	IN	A	185.199.110.153
hourstorycomic.github.io. 3546	IN	A	185.199.108.153
hourstorycomic.github.io. 3546	IN	A	185.199.111.153
```

**Comment by ourstorycomic** [comment]
no actually it's not "hourstorycomic.github.io." lmfao, it's "ourstorycomic.github.io."

**Comment by tarampampam** [comment]
Make a new PR with all required changes, please

**Comment by tarampampam** [comment]
Oh, it was my mistake, one second

**Comment by tarampampam** [comment]
Updated:

```
$ dig @1.1.1.1 +noall +answer rwbox.is-an.app
rwbox.is-an.app.	300	IN	CNAME	ourstorycomic.github.io.
ourstorycomic.github.io. 3600	IN	A	185.199.110.153
ourstorycomic.github.io. 3600	IN	A	185.199.111.153
ourstorycomic.github.io. 3600	IN	A	185.199.109.153
ourstorycomic.github.io. 3600	IN	A	185.199.108.153
```

**Pull request merged**